lib/ansible/module_utils/facts.py

```diff
@@ -604,6 +604,10 @@ class Distribution(object):
     This is unit tested. Please extend the tests to cover all distributions if you have them available.
```

Comment:
Anybody knows if we could make the existing test case to prevent this on other distros like arch linux or Alpine ?